src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts

```diff
@@ -97,6 +97,11 @@ export interface IInlineChatWidgetConstructionOptions {
 	 * The men that rendered in the lower right corner, use for feedback
 	 */
 	feedbackMenuId: MenuId;
+
+	/**
+	 * Whether the code blocks are editable.
+	 */
+	editableCodeBlock?: boolean;
```

Comment:
This makes all code blocks editable which IMO is too coarse-grained. Could this be somehow controlled per rendered code block? Maybe via the assoicated language or resource of a codeblock? As a stepping stone this boolean could be passed when calling `updateChatMessage` but ultimatively this should be per code-block instance